src/main/java/com/iota/iri/conf/ProtocolConfig.java

```diff
@@ -43,7 +41,6 @@
         String REQUEST_HASH_SIZE = "The size of the requested hash in a packet. Its size is derived from the minimal MWM value the network accepts. The larger the MWM -> the more trailing zeroes we can ignore -> smaller hash size.";
         String P_DROP_TRANSACTION = DescriptionHelper.PROB_OF + "dropping a received transaction. This is used only for testing purposes.";
         String P_SELECT_MILESTONE = DescriptionHelper.PROB_OF + "requesting a milestone transaction from a neighbor. This should be a large since it is imperative that we find milestones to get transactions confirmed";
```

Comment:
Since you deleted the line below, makes sense to remove this as well

Reply:
part of #30